### PR TITLE
Kubernetes Tutorial Added

### PR DESCRIPTION
**Pull request by arshiagg**
Kubernetes version 1.6
Contiv Tutorial added to website.

### CODE REVIEW

**Comment by gaurav-dalvi** [comment]
@arshiagg  this did not work on my macbook.

```
 make demo-kubeadm
BUILD_VERSION=1.0.3 make cluster-kubeadm
cd cluster && vagrant destroy -f && \
	vagrant up kubeadm-master && \
	vagrant up kubeadm-worker0
==> kubeadm-worker0: VM not created. Moving on...
==> kubeadm-master: VM not created. Moving on...
==> swarm-mode-worker0: VM not created. Moving on...
==> swarm-mode-master: VM not created. Moving on...
==> legacy-swarm-worker0: VM not created. Moving on...
==> legacy-swarm-master: VM not created. Moving on...
Bringing machine 'kubeadm-master' up with 'virtualbox' provider...
There are errors in the configuration of this machine. Please fix
the following errors and try again:

vm:
* A box must be specified.


```

**Comment by dseevr** [comment]
@gaurav-dalvi we saw this error on @arshiagg's laptop and fixed it by wiping out all her existing VMs

I'm not sure what's going on there, but all the code paths in `cluster/Vagrantfile` result in `vm.box` being set...

**Comment by gaurav-dalvi** [comment]
@dseevr : exact same. I dont have any vagrant box on my machine.

**Comment by gaurav-dalvi** [comment]
Great work @arshiagg . I have verified this and It worked smoothly.

File: websrc/source/documents/tutorials/container-101.md [hunk 1]
```diff
@@ -321,24 +324,24 @@ There are two main container networking models discussed within the community.
...
 
-CNM (Container Network Model) is Docker's libnetwork network model for containers
-- An endpoint is container's interface into a network
-- A network is collection of arbitrary endpoints
-- A container can belong to multiple endpoints (and therefore multiple networks)
-- CNM allows for co-existence of multiple drivers, with a network managed by one driver
-- Provides Driver APIs for IPAM and Endpoint creation/deletion
-- IPAM Driver APIs: Create/Delete Pool, Allocate/Free IP Address
-- Network Driver APIs: Network Create/Delete, Endpoint Create/Delete/Join/Leave
+CNM (Container Network Model) is Docker's libnetwork network model for containers
```
Comment (dseevr) [inline websrc/source/documents/tutorials/container-101.md]: What's up with the extra whitespace here?  Was this automatically added by your editor?

You can edit this file and amend the commit and force push it (good opportunity to practice 😄)

Comment (dseevr) [inline websrc/source/documents/tutorials/container-101.md]: ok or don't bother lol

Comment (arshiagg) [inline websrc/source/documents/tutorials/container-101.md]: @dseevr extra whitespace? like the two spaces at the end of each line? In markdown, two spaces after a line means new line :)

Comment (dseevr) [inline websrc/source/documents/tutorials/container-101.md]: huh, TIL "  " becomes "\<br\>"

**Pull request merged**